ssas/service/public/api.go

```diff
@@ -55,6 +54,13 @@ type PasswordRequest struct {
 	Password string `json:"password"`
 }
 
+type SystemResponse struct {
+	ClientID		string `json:"client_id"`
+	ClientSecret	string `json:"client_secret"`
```

Comment:
Why does GitHub not treat tabs the same way our IDE's do?

Reply:
What happens if you run `go fmt` on the file?